### PR TITLE
Pincode fixes

### PR DESCRIPTION
**Pull request by gHashTag**
[#296](https://github.com/haqq-network/haqq-wallet/issues/296)

### CODE REVIEW

File: src/components/pin.tsx [hunk 1]
```diff
@@ -111,7 +115,11 @@ export const Pin = forwardRef(
         <Text t4 style={page.title}>
           {title}
         </Text>
-        {error && <Text clean>{error}</Text>}
+        {error && (
+          <Text t6 style={page.error}>
```
Comment (vivalaakam) [inline src/components/pin.tsx:119]: maybe you mean `t10` and don't overwrite fontSize and lineHeight in styles?

File: src/screens/onboarding-repeat-pin.tsx [hunk 2]
```diff
@@ -73,7 +73,7 @@ export const OnboardingRepeatPinScreen = () => {
           <View style={[page.dot, pin.length >= 5 && page.active]} />
           <View style={[page.dot, pin.length >= 6 && page.active]} />
         </View>
-        <Text clean style={page.error}>
+        <Text t6 style={page.error}>
```
Comment (vivalaakam) [inline src/screens/onboarding-repeat-pin.tsx:76]: maybe you mean t10 and don't overwrite fontSize and lineHeight in styles?

File: src/components/ui/error-text.tsx [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+      ]}
+      {...props}
+    />
+  );
+};
+
+const page = StyleSheet.create({
+  e0Style: {
+    fontStyle: 'normal',
+    fontSize: 16,
```
Comment (vivalaakam) [inline src/components/ui/error-text.tsx:43]: what is the fontSize and lineHeight for here?

Comment (gHashTag) [reply src/components/ui/error-text.tsx:43]: This is due to the fact that text t6 uses the required text font and weight, but the line height and font size are not suitable
image

Comment (vivalaakam) [reply src/components/ui/error-text.tsx:43]: there should not be a font size and line height, all these parameters should be described in the text, if there are discrepancies, then you need to ask the design to update everything in itself and, if necessary, add a new key to the text component

**Review by vivalaakam — APPROVED** [review]

**Pull request merged**